vacs-map-app/src/components/MapContainerColor.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <BaseMap>
+    <template v-slot="{ map, mapReady }">
+      <GridSource :id="sourceId" :map="map" :mapReady="mapReady" />
+      <GridOverlay
+        id="grid-layer-1"
+        :color-column="selectedColumn"
+        :color-column-extent="selectedColumnExtent"
+        :color-column-quintiles="selectedColumnQuintiles"
+        :color-diverging="false"
+        :sourceId="sourceId"
+        :map="map"
+        :mapReady="mapReady"
+      />
```

Comment:
This is the layer we add to the map. I'd love to make this layer as flexible as we can (you can see in other maps that the props can vary a lot). But if we get to a point where we're making very specific changes to the layer that we don't want to include on all layers, we should consider having multiple components instead.